apps/events/models.py

```diff
@@ -232,6 +232,7 @@ class ScheduledMessage(BaseTeamModel):
     last_triggered_at = models.DateTimeField(null=True)
     total_triggers = models.IntegerField(default=0)
     is_complete = models.BooleanField(default=False)
+    extra_data = models.JSONField(blank=True, default=dict)
```

Comment:
I was going to suggest making this a cron expression for future flexibility but that seems overkill.

I do think it should be renamed to `custom_schedule_params` or something to give it more meaning.

Reply:
This confirms it then. I was between having a dedicated field vs an `extra_data` field that we just happen to use for the preferred schedule. I'll update :+1: